improver_tests/acceptance/test_extract.py

```diff
@@ -149,3 +149,60 @@ def test_range_constraints(tmp_path):
     ]
     run_cli(args)
     acc.compare(output_path, kgo_path, recreate=False)
+
+
+def test_spot(tmp_path):
+    """Test extraction from spot file"""
+    kgo_dir = acc.kgo_root() / "extract/sites"
+    kgo_path = kgo_dir / "kgo_spot.nc"
+    input_path = kgo_dir / "input_spot.nc"
+    output_path = tmp_path / "output.nc"
+    args = [
+        input_path,
+        "--constraints",
+        "wmo_id=['700', '845', '996', '3346', '3382']",
+        "--output",
+        output_path,
+    ]
+    run_cli(args)
+    acc.compare(output_path, kgo_path)
+
+
+def test_uk_grid(tmp_path):
+    """Test subsetting of UK standard gridded data"""
+    kgo_dir = acc.kgo_root() / "extract/grids"
+    kgo_path = kgo_dir / "kgo_grid_uk.nc"
+    input_path = kgo_dir / "input_grid_uk.nc"
+    grid_spec_path = kgo_dir / "grid_spec.json"
```

Comment:
This variable is unused, both here and on line 196.

Reply:
Ah, thanks! This was a remnant from the previous `test_subset_cube`